includes/admin/class-wc-rest-payments-terminal-locations-controller.php

```diff
@@ -121,8 +121,8 @@ public function get_store_location( $request ) {
 		);
 
 		// If address is not populated, emit an error and specify the URL where this can be done.
-		$is_address_unpopulated = 2 === count( $location_address ) && isset( $location_address['country'], $location_address['state'] );
-		if ( $is_address_unpopulated ) {
+		$is_address_populated = isset( $location_address['country'], $location_address['city'], $location_address['postal_code'], $location_address['line1'] );
```

Comment:
This is extremely strict. E.g. https://tosbourn.com/list-of-countries-without-a-postcode/

Reply:
We are currently supporting US only, but I added a corresponding note in 48594dd9ddf3667e069e39da71b0a46f1137d441